app/assets/javascripts/charts.js

```diff
@@ -58,6 +58,8 @@ function resize_label(label){
 }
 
 function expanded_pie(target, data){
+    var max={data:0}, sum = 0;
```

Comment:
Indentation is off here, and also in a few cases below - please match the existing style, which is two spaces.